src/mongoose_http_client.erl

```diff
@@ -44,6 +43,16 @@
 
 -spec start() -> ok.
 start() ->
+    case ets:info(?MODULE) of
+        undefined ->
+            Heir = case whereis(ejabberd_sup) of
```

Comment:
This trick is already used in other places in the code. It'd be good to unify this before we have too many incarnations of it.

Reply:
There is an internal ticket for this. :)